manifests/s/Streetwriters/Notesnook/3.0.0/Streetwriters.Notesnook.installer.yaml

```diff
@@ -0,0 +1,29 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Streetwriters.Notesnook
+PackageVersion: 3.0.0
+InstallerLocale: en-US
+UpgradeBehavior: install
+Commands:
+- notesnook
+ReleaseDate: 2024-04-29
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/streetwriters/notesnook/releases/download/v3.0.0/notesnook_win_x64.exe
+  InstallerSha256: 4A0CF56A8E4B5C7103A607431768135C60DD2379763F60996A34AFA729320CB1
+- Architecture: x64
+  InstallerType: portable
+  InstallerUrl: https://github.com/streetwriters/notesnook/releases/download/v3.0.0/notesnook_win_x64.exe
+  InstallerSha256: 4A0CF56A8E4B5C7103A607431768135C60DD2379763F60996A34AFA729320CB1
+- Architecture: arm64
+  InstallerType: nullsoft

```

Comment:
This same installer has `portable` above and `nullsoft` here - is this a multi-type installer? This error happened too early in the process to be completed through manual validation, so I'm not quite sure how to proceed.